#Name of the detected Object
string object_name

#Rotated Rectangle of the source image matching the detected object
RotatedRect rrect
BoundingBox bbox